cloud_utils/slack_utils.py

```diff
@@ -1,7 +1,14 @@
+import logging
+import os
 import traceback
-from typing import Dict
+from typing import Callable, Dict
 
 import gamla
+import slack_sdk
+from gamla import frozendict
+from slack_sdk import errors as slack_errors
+
+_SLACK_NOTIFICATIONS_WEBHOOK_URL = os.getenv("SLACK_NOTIFICATIONS_WEBHOOK")
```

Comment:
Please do not use os.getenv but os.environ[]
We want the code to fail if the env var is not configured.

Reply:
Also, let's move this configuration to the caller (i.e. replay)